CommandLine.c

```diff
@@ -177,9 +177,15 @@ static CommandLineSettings parseArguments(const char* program, int argc, char**
 
             if (!username) {
                flags.userId = geteuid();
-            } else if (!Action_setUserOnly(username, &(flags.userId))) {
-               fprintf(stderr, "Error: invalid user \"%s\".\n", username);
-               exit(1);
+            } else if (Action_setUserOnly(username, &(flags.userId))) {
+               /* Assigned by Action */
+            } else {
```

Comment:
Why not keep the original negated check?
```suggestion
            } else if (!Action_setUserOnly(username, &(flags.userId))) {
```

Reply:
left over from v1, I guess. fixed